UKHomeOffice/cerberus-service
# Cop 10921: Add test to Forward RORO movement records from cop targeting api to cerberus workflow service

## Description
Add test to Forward RORO movement records from cop targeting api to cerberus workflow service
## To Test
npm run cypress:test -- --spec cypress/integration/cerberus/create-tasks.spec.js

Should create a Roro-Accompanied task from Cop-targetting API and forward to Cerberus workflow service
Should create a Roro-Unaccompanied task from Cop-targetting API and forward to Cerberus workflow service
Should create a Roro-Tourist task from Cop-targetting API and forward to Cerberus workflow service
## Developer Checklist

\* Required
